lib/maid/watch.rb

```diff
@@ -6,7 +6,14 @@ class Maid::Watch
 
   def initialize(maid, path, options = {}, &rules)
     @maid = maid
-    @options = options
+    if options.nil? 
+      @lazy = true
+      @options = { wait_for_delay: 10, 
+                   ignore: [/\.crdownload$/, /\.download$/, /\.aria2$/, /\.td$/, /\.td.cfg$/, /\.part$/] }
```

Comment:
It seems like we need to refactor to enable this.  Moving the regexps to some kind of data structure would probably involved.

For example:

```ruby
def downloading?(path)
  downloading_file_regexps.any? { |re| path.match(re) }
end

private

def downloading_file_regexps
  [/\.crdownload$/, /\.download$/, /\.aria2$/, /\.td$/, /\.td.cfg$/, /\.part$/]
end
```

That doesn't quite capture all of it, but it's close.